SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestEntityInteration/EntityInteractionTest.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EntityInteractionTest.h"
+#include "EntityInteractionTestActor.h"
+#include "Kismet/GameplayStatics.h"
+
+#include "SpatialGDKSettings.h"
+
+/**
+ * This test goes through the various methods we have added to handle remote action and interaction between Actors.
+ * It tests two things :
+ * - That all calls eventually make their way to the target actor
+ * - That calls that are expected to short circuit do so, and the one that are not results in a remote call.
+ * The map is setup with two workers, each with auth over two actors. Both workers run symmetrical code.
+ * So for each RPC emitted, the symmetrical actor is expected to emit a similar one.
+ * This is how we test that the "delayed" RPCs have made their way through the network.
+ * This test is only relevant when the routing worker is enabled in the GDK settings.
```

Comment:
Is there any value in running this test without the routing worker enabled? If not, can we add this test to a map that mandates `CrossServerRPCImplementation` is set correctly?

Reply:
There is none, it just goes through a dummy step.
How would I make the map require the right settings ?

Reply:
Fixed, but in its own map with the correct settings.